openid-connect-4-credential-issuance-1_0.md

```diff
@@ -469,13 +619,12 @@ A Client makes a Credential Request by presenting the following parameters:
 * `format`: OPTIONAL. Format of the credential to be issued. If not present, the issuer will determine the credential 
 format based on the clients format default. 
 * `sub_jwk`: CONDIIONAL. the key material the new credential shall be bound to.  
-* `did`: CONDIIONAL. the DID the credential shall be bound to. `sub_jwk` and `did` are mutual exclusive. 
+* `did`: CONDIIONAL. the DID the credential shall be bound to. `sub_jwk` and `did` are mutually exclusive. 
 * `proof` CONDITIONAL. JSON Object containing the proof of possession of the key material the issued credential shall be 
 bound to. The `proof` structure depends on the proof type. At the minimum, the following parameters MUST be included:
```

Comment:
we need to make it clear that there are three options, provide
- only sub_jwk
- only did
- did and proof

Reply:
clarify that other claims might be included in proof

Reply:
this is minimum we should describe in proof is how to sign the binding material.